src/aquarius/Aquarius.ts

```diff
@@ -215,6 +215,51 @@ export class Aquarius {
         return this.retrieveDDO(undefined, metadataServiceEndpoint)
     }
 
+    /**
+     * Transfer ownership of a DDO
+     * @param  {DID | string} did DID of the asset to update.
+     * @param  {String} newOwner New owner of the DDO
+     * @param  {String} updated Updated field of the DDO
+     * @param  {String} signature Signature using updated field to verify that the consumer has rights
+     * @return {Promise<String>} Result.
+     */
+    public async transferOwnership(
+        did: DID | string,
+        newOwner: string,
+        updated: string,
+        signature: string
+    ): Promise<string> {
+        did = did && DID.parse(did)
+        const fullUrl = `${this.url}${apiPath}/owner/update//${did.getDid()}`
```

Comment:
is the double slash required?

Otherwise:
```suggestion
        const fullUrl = `${this.url}${apiPath}/owner/update/${did.getDid()}`
```

Reply:
fixed, thanks